CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -27,7 +27,10 @@
 #include <CoreFoundation/ForFoundationOnly.h>
 #include <fts.h>
 #include <pthread.h>
+
+#if !TARGET_OS_ANDROID
```

Comment:
Would using the other syntax of `#if __has_include(<execinfo.h>)` work here, to keep it in line with the other tests?